examples/sweep-regularization.py

```diff
@@ -43,31 +42,34 @@
 )
 
 # Setup a single writer and viewer
-writer = WaveorderWriter("./output")
-writer.create_zarr_root("reconstructions_" + timestamp)
+reg_powers = 10.0 ** np.arange(-3, 3)
+
+dataset = open_ome_zarr(
+    "./output/reconstructions_" + timestamp,
+    layout="fov",
+    mode="w",
+    channel_names=[f"{reg:.1e}" for reg in reg_powers],
+)
+
+dataset.create_zeros(
+    "0", shape=(1, len(reg_powers), Z, Y, X), dtype=np.float32
+)
+
 v = napari.Viewer()
 v.add_image(data)
 
 # Loop through regularizations
-reg_powers = np.arange(-3, 3)
-for i, reg_power in enumerate(reg_powers):
-    reg = 10.0**reg_power
-    print(f"Reconstructing with 3D phase with reg = {reg:.1e}")
+for i, reg in enumerate(reg_powers):
+    print(f"Reconstructing with 3D phase with reg = {reg}")
 
     phase3D = reconstruct_phase3D(
         data, reconstructor, method="Tikhonov", reg_re=reg
     )
 
     # Save each regularization into a "position" of the output zarr
-    writer.init_array(
-        position=i,
-        data_shape=(1, 1, Z, Y, X),
-        chunk_size=(1, 1, 1, Y, X),
-        chan_names=["Phase"],
-    )
-    writer.write(phase3D, p=i, t=0, c=0, z=slice(0, Z))
+    dataset["0"][0, i, ...] = phase3D
```

Comment:
Before I was writing to multiple positions, but now I'm writing to channels (only because that's what seemed easiest). 

Should I change to 'hcs' mode? Or is there an easier way?

Reply:
I think storing in channels makes sense in that they are different observations of the same FOV.